action.yml

```diff
@@ -4,7 +4,7 @@
 # We also define metadata here:
 # See: https://help.github.com/en/articles/metadata-syntax-for-github-actions
 
-name: 'docker-image-size-limit'
+name: 'docker-image-size-limit2'
```

Comment:
```suggestion
name: 'docker-image-size-limit'
```

Reply:
Done in https://github.com/wemake-services/docker-image-size-limit/commit/b76c142f89ffe291e1c800b1daa2c6ebf46a2e4f